internal/observability/event/eventer_config.go

```diff
@@ -13,6 +13,7 @@ type EventerConfig struct {
 	ObservationsEnabled bool          `hcl:"observations_enabled"` // ObservationsEnabled specifies if observation events should be emitted.
 	SysEventsEnabled    bool          `hcl:"sysevents_enabled"`    // SysEventsEnabled specifies if sysevents should be emitted.
 	Sinks               []*SinkConfig `hcl:"-"`                    // Sinks are all the configured sinks
+	ErrorEventsDisabled bool          `hcl:"-"`                    // ErrorEventsDisabled will disable error events from being emitted.  This should only be used to turn off error events in tests.
```

Comment:
Why is this the inverse of the other options? I assume it's because we want a different default behavior, but I'd prefer if this was the same as the others and we just initialized it explicitly to the default. I worry with this inversion that someone could accidentally configure this or one of the others to the opposite of what they want. What do you think?

Reply:
Of course, I did this intentionally.  We don't want anyone configuring them and I wanted to increase the load on anyone trying to fool with setting error events.